src/mainviews/job-detail.tsx

```diff
@@ -106,206 +104,190 @@ export function JobDetail(props: IJobDetailProps): JSX.Element {
-              </Stack>
-            </AccordionDetails>
-          </Accordion>
-        </>
-      );
-    }
-  }
+              </Grid>
+            </React.Fragment>
+          ))}
+        </Grid>
+      </CardContent>
+    </Card>
+  );
 
-  return (
-    <>
-      <Box sx={{ maxWidth: '500px', p: 4 }}>
+  const OtherOptions = () => (
+    <Card>
+      <CardContent>
         <Stack spacing={4}>
-          <div role="presentation">
-            <Breadcrumbs aria-label="breadcrumb">
-              <Link
-                underline="hover"
-                color="inherit"
-                onClick={(
-                  _:
-                    | React.MouseEvent<HTMLAnchorElement, MouseEvent>
-                    | React.MouseEvent<HTMLSpanElement, MouseEvent>
-                ): void => props.setView('ListJobs')}
-              >
-                {trans.__('Notebook Jobs')}
-              </Link>
-              <Typography color="text.primary">{props.model.jobId}</Typography>
-            </Breadcrumbs>
-          </div>
-          <Heading level={1}>{trans.__('Job Detail')}</Heading>
-          <MainArea />
+          <TextFieldStyled
+            label={trans.__('Idempotency token')}
+            defaultValue={job?.idempotency_token ?? ''}
+          />
+          <FormLabel component="legend">{trans.__('Tags')}</FormLabel>
+          {job?.tags &&
+            job?.tags.map((tag, idx) => (
+              <TextFieldStyled
+                key={tag}
+                label={trans.__('Tag %1', idx + 1)}
+                defaultValue={tag}
+              />
+            ))}
+        </Stack>
+      </CardContent>
+    </Card>
+  );
+
+  const AdvancedOptions = () => (
+    <Card>
+      <CardContent>
+        <Stack component="form" spacing={4}>
+          <FormLabel component="legend">
+            {trans.__('Advanced Options')}
+          </FormLabel>
+          <props.advancedOptions
+            jobsView={'JobDetail'}
+            model={props.model}
+            handleModelChange={model => {
+              return;
+            }}
+            errors={{}}
+            handleErrorsChange={errors => {
+              return;
+            }}
+          />
         </Stack>
-      </Box>
-    </>
+      </CardContent>
+    </Card>
+  );
+
+  const BreadcrumbStyled = () => (
+    <div role="presentation">
+      <Breadcrumbs aria-label="breadcrumb">
+        <Link
+          underline="hover"
+          color="inherit"
+          onClick={(
+            _:
+              | React.MouseEvent<HTMLAnchorElement, MouseEvent>
+              | React.MouseEvent<HTMLSpanElement, MouseEvent>
+          ): void => props.setView('ListJobs')}
+        >
+          {trans.__('Notebook Jobs')}
+        </Link>
+        <Typography color="text.primary">
+          {job?.name ?? props.model.jobId}
+        </Typography>
+      </Breadcrumbs>
+    </div>
+  );
+
+  return (
+    <Box sx={{ p: 4 }} style={{ height: '100%', boxSizing: 'border-box' }}>
```

Comment:
Do not set the `height` property here. This is causing the bottom padding to not be applied to the list view body.

Reply:
Fixed in latest commit.